blog/_posts/2024-08-07-eks-to-aks-series-part1.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Mastering the Move: EKS to AKS by Example"
+description: "Learn how to migrate Amazon EKS workloads to Azure AKS using KEDA and Karpenter in this comprehensive guide."
+date: 2050-08-24
```

Comment:
Update this date to something today or earlier. This is used to render the page. Ideally the file name and this date match